test/display_test.py

```diff
@@ -65,20 +65,53 @@ def todo_test_flip(self):
 
         self.fail()
 
-    def todo_test_get_active(self):
+    def test_get_active(self):
+        """Test the get_active function"""
+
+        #Initially, the display is not active
+        pygame.quit()
+        pygame.display.quit()
+        self.assertEqual(pygame.display.get_active(), False)
+
+        #get_active defaults to true after a set_mode
+        pygame.init()
+        pygame.display.set_mode((640, 480))
+        self.assertEqual(pygame.display.get_active(), True)
+
+        #get_active after init/quit should be False
+        #since no display is visible
+        pygame.display.quit()
+        pygame.display.init()
+        self.assertEqual(pygame.display.get_active(), False)
+
+
+    @unittest.skipIf(
+        os.environ.get("SDL_VIDEODRIVER") == "dummy",
+        'requires the SDL_VIDEODRIVER to be a non dummy value'
+    )
+    def test_get_active_iconify(self):
+        """Test the get_active function after an iconify"""
+
+        pygame.init()
```

Comment:
should delete this as `setUp()` will already have called `pygame.display.init()` for us.